# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/basic/bug1731540.js
# Commit: c71e5fccc8e6
# Full Hash: c71e5fccc8e6b2816158a1afb531a510105f174f
# Author: Iain Ireland <iireland@mozilla.com>
# Date: 2021-09-22 04:09:13
# Description:
#   Bug 1731540: Handle empty iterators when unwinding uncatchable exceptions r=jandem
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D126125
# ==============================================================================

diff -r 000302929625 -r c71e5fccc8e6 js/src/jit-test/tests/basic/bug1731540.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/basic/bug1731540.js	Tue Sep 21 17:01:33 2021 +0000
@@ -0,0 +1,4 @@
+// |jit-test| exitstatus: 6
+v11 = undefined;
+interruptIf(true);
+for (v63 in v11);